apps/frontend/app/(client)/(main)/course/[courseId]/_components/Sidebar.tsx

```diff
@@ -0,0 +1,68 @@
+'use client'
+
+import { cn } from '@/libs/utils'
+import assignmentIcon from '@/public/icons/assignment.svg'
+import examIcon from '@/public/icons/exam.svg'
+import homeIcon from '@/public/icons/home.svg'
+import noticeIcon from '@/public/icons/notice.svg'
+import type { Route } from 'next'
+import Image from 'next/image'
+import Link from 'next/link'
+import { useSearchParams } from 'next/navigation'
+
+export function Sidebar() {
+  const searchParams = useSearchParams()
+  const courseId = searchParams.get('courseId')
+
+  const navItems: { name: string; path: Route; icon: string }[] = [
+    { name: 'Home', path: `/course/${courseId}` as Route, icon: homeIcon },
```

Comment:
```suggestion
    { name: 'Home', path: `/course/${courseId}` as const, icon: homeIcon },
```

const 단언문으로 바꿔주세요!

https://www.notion.so/skkuding/f825c8cdc9b74598bd00dadadf58815d?pvs=4#173ef9cff5458024b137cb91743c5007

Reply:
너무 친절하세요!